src/routes/github/github-router.ts

```diff
@@ -25,36 +39,35 @@ subRouter.post("/webhooks",
 	returnOnValidationError,
 	WebhookReceiverPost);
 
-//Have an cover all middleware to extract the optional gitHubAppId
-//subRouter.use(param("uuid").isUUID('all'), GithubServerAppMiddleware);
+// Create-branch is seperated above since it currently relies on query param to extract the jirahost
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
 
-// TODO: use jiraSymmetricJwtMiddleware and GithubServerAppMiddleware for all paths below once "create branch" supports JWT
-// subRouter.use(jiraSymmetricJwtMiddleware);
-// subRouter.use(GithubServerAppMiddleware);
+// OAuth Routes
+subRouter.use(GithubOAuthRouter);
+
+subRouter.use(jiraSymmetricJwtMiddleware);
+subRouter.use(GithubServerAppMiddleware);
 
 // OAuth Routes
-subRouter.use(jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubOAuthRouter);
+subRouter.use(GithubOAuthRouter);
 
 // CSRF Protection Middleware for all following routes
 subRouter.use(csrfMiddleware);
 
-subRouter.use("/setup", jiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubSetupRouter);
+subRouter.use("/setup", GithubSetupRouter);
 
 // TODO: use GithubAuthMiddleware for all paths below once "create branch" supports JWT
```

Comment:
Maybe remove this comment? Tbh I don't understand it.

Reply:
ahhhhh this comment, snuck back in!  deleting